cinema/views.py

```diff
@@ -124,6 +160,13 @@ class OrderViewSet(viewsets.ModelViewSet):
     )
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (OrderPermission,)
```

Comment:
u can use here built-in permission class (IsAuthenticated) also u can use generic and mixins to chose need actions, just like this 
class OrderViewSet(
    mixins.CreateModelMixin,
    mixins.ListModelMixin,
    viewsets.GenericViewSet
):